tools/cis/prep_cis.py

```
@@ -31,7 +31,7 @@
     title = tab.title
     if title == "License for Use":
         library_copyright = tab["B11"].value + "\n" + tab["B13"].value
-    elif title == "Controls V8":
+    elif title.startswith("Controls V"): """The script ignored the controls sheet because the title wasn't an exact match in the latest version of the CIS Controls .xlsx file. The title currently reads Controls V8.1.2 instead of Controls V8. Initial workaround as pointed out by my colleague Jak, was to rename the sheet to Controls V8. The fix I applied was to use the startswith function to prompt the script to parse the sheet as long as the title start with Controls V. This should accommodate all version changes going forward."""
```

Ruff (0.8.2)

Multiple statements on one line (colon)

(E701)